htdocs/PHP/internal_libraries/languages/mail_content_fr.php

```diff
@@ -0,0 +1,20 @@
+<?php
+
+/**
+* Translations for email notifications
+* Language: French
+*/
+
+$translation = array();
+
+// Access request email notification
+$translation["access_request_subject"] = "Demande d'accès aux notes d'urgence";
+$translation["access_request_html_content"] = "<p>Bonjour,</p>
+                  <p>Quelqu'un essaie d'accéder à votre note d'urgence dont l'objet est le suivant: <b>{$subject}</b></p>
+                  <p>Si vous voulez bloquer temporairement l'accès à cette note pendant 60 jours, vous devez cliquer sur le lien ci-dessous:</p>
+                  <p><b><a href='{$base_path}/block.php?m={$hashed_note}' target='_blank'>Oui, je veux bloquer temporairement l'accès à cette note pendant 60 jours.</a></b></p>
+                  <hr>
+                  <p>Vous avez des questions ou besoin d'aide ? Écrivez-nous à <a href='mailto:{$SMTP_username}'>{$SMTP_username}</a></p>";
+$translation["access_request_non-html_content"] = "Bonjour, quelqu'un essaie d'accéder à votre note d'urgence qui a le sujet suivant: {$subject}. Si vous souhaitez bloquer temporairement l'accès à cette note pendant 60 jours, vous devez visiter l'URL suivante sur votre navigateur: {$base_path}/block.php?m={$hashed_note}. Vous avez des questions ou besoin d'aide ? Écrivez-nous à {$SMTP_username}";
```

Comment:
- Does "Bonjour" in French work also as a way to say "Hi" at any time of the day? I’m a bit worried if a user receives the email notification during the night. In Italian can be used only between a certain time in the morning, I’m not sure if in French works in a similar way.

- Thanks to your translations I discovered that the French punctuation requires a space before double signed punctuations marks but I don’t know if this is a particular grammar case that doesn’t need a space before ":".

Reply:
> Does "Bonjour" in French work also as a way to say "Hi" at any time of the day?

"Bonjour" is the equivalent of "Hello", it works anytime in the day. "Hi" can be translated as "Salut", but it is a formulation that you usually use with friends and family, not with people you do not really know. "Bonjour" works everywhere with everyone.

> I don’t know if this is a particular grammar case that doesn’t need a space before ":".

Indeed, in french grammar we have to put a space before ":", but I sometimes mix things with english :grimacing: I'll fix this!